migrator/runner/all.go

```diff
@@ -14,6 +14,7 @@ import (
 	_ "github.com/stackrox/rox/migrator/migrations/m_109_to_m_110_networkpolicy_guidance_2"
 	_ "github.com/stackrox/rox/migrator/migrations/m_110_to_m_111_replace_deprecated_resources"
 	_ "github.com/stackrox/rox/migrator/migrations/m_111_to_m_112_groups_invalid_values"
+	_ "github.com/stackrox/rox/migrator/migrations/m_168_to_m_169_postgres_remove_clustercve_permission"
```

Comment:
Do we want to pull this import into its own section of postgres -> postgres migrations in a way similar to the rocks -> postgres migrations are in their own section? We could start a new section at the top, which would also give us an opportunity to leave a descriptive message describing why there's a number skip and that the new particular section is postgres -> postgres only